### DISCUSSION TITLE
need to buttons in shifting to next excalidraw shapes

### DISCUSSION THREAD

**Discussion by sohailamjad256** [excalidraw/excalidraw#4354] [Q&A]
cant we give next buttons and previous buttons to see different library item shapes instead of showing big popup window in library items ?? please help  **!**important***
![Screenshot (99)](https://user-images.githubusercontent.com/53993286/144367743-a164ede5-0f88-476d-a3f9-bd48df218565.png)

**Comment by dwelle** [comment]
locking in favor of https://github.com/excalidraw/excalidraw/issues/4352
